x/session/keeper/session_hydrator.go

```diff
@@ -96,7 +96,9 @@ func (k Keeper) hydrateSessionMetadata(ctx sdk.Context, sh *sessionHydrator) err
 	}
 
 	sh.session.NumBlocksPerSession = NumBlocksPerSession
-	sh.session.SessionNumber = int64(sh.blockHeight / NumBlocksPerSession)
+	sh.session.SessionNumber = sh.blockHeight / NumBlocksPerSession
+
+	// NB: SessionStartBlockHeight should be aligned to NumBlocksPerSession.
```

Comment:
```suggestion
	// TODO_BLOCKER: SessionStartBlockHeight should be aligned to NumBlocksPerSession.
```

Reply:
Once `NumBlocksPerSession` is a governance param, we'll be able to compute it, even if the param itself is updated mid session.